schematic_api/Dockerfile

```diff
@@ -51,7 +77,15 @@ COPY  schematic_api ./schematic_api
 
 # copy great_expectations folder
 COPY great_expectations ./great_expectations
-RUN chown -R www-data:www-data /opt/app/great_expectations/
+
+# copy tests folder because some endpoints by default download to the tests folder
+COPY tests ./tests
+
+# change permission
+RUN chown -R www-data:www-data ${APP_DIR}
+
+# allow downloading to synapse cache 
+RUN chown -R www-data:www-data /root
```

Comment:
Why is the synapse cache under `/root`? Can we configure it to be in another location, maybe `/var/tmp/synapse`?

Reply:
@linglp It's probably not a big deal in our case, but `/root` is the home directory of the root user, and is not really an appropriate place for services to store data. See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html for more info on the standard linux filesystem hierarchy.

And if its the default directory, I guess that means we're running the service as the root user (which can be a potential security issue). If that is the case, we should eventually change the container to run as a service user instead of the root user, but that can be a challenging migration that I don't want to block this PR on. Here's a short article I found on that: https://www.howtogeek.com/devops/why-processes-in-docker-containers-shouldnt-run-as-root/

Reply:
I agree with the proposal of @jesusaurus. Installing and running the app as a non-root user is more secure. This work should be tracked in a separate ticket.

Reply:
Thanks both. I created an issue here: https://sagebionetworks.jira.com/browse/DCA-173 to keep track of this issue.